.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: python
 
+sudo: required
```

Comment:
`sudo` is no longer needed: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Reply:
Thanks @hugovk! Do you want to make a PR for it?

Reply:
Sure, please see PR https://github.com/pydap/pydap/pull/193!